app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx

```diff
@@ -33,19 +33,19 @@ export default function ContactTable({
                 }}
               />
             </th>
-            <th className="text-left text-gray-500">Name</th>
-            <th className="text-left text-gray-500"> Email</th>
-            <th className="text-center text-gray-500">Department</th>
-            <th className="text-center text-gray-500">Contact Number</th>
-            <th className="text-center text-gray-500">Add to Basecamp</th>
+            <th className="text-left">Name</th>
+            <th className="text-left"> Email</th>
+            <th className="text-center">Department</th>
+            <th className="text-center">Contact Number</th>
+            <th className="text-center">Add to Basecamp</th>
             <th></th>
           </tr>
         </thead>
         <tbody>
           {contacts.map(contact => (
             <tr
               key={contact.id}
-              className={"group cursor-pointer bg-white hover:bg-gray-50"}
+              className={"cursor-pointer bg-white hover:bg-gray-50"}
```

Comment:
Please remove this, `nui-table--hover` adds hover transition effects.

Reply:
Removed unwanted css classes.